Snapcraft Test Errors

https://github.com/freeorion/freeorion/actions/runs/3902142804/jobs/6664843226

````
Err:1 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main amd64 libegl-mesa0 amd64 22.0.5-0ubuntu0.1
  404  Not Found [IP: 40.119.46.219 80]
Get:2 http://azure.archive.ubuntu.com/ubuntu jammy/main amd64 libegl1 amd64 1.4.0-1 [28.6 kB]
Get:3 http://azure.archive.ubuntu.com/ubuntu jammy/main amd64 libgles2 amd64 1.4.0-1 [18.0 kB]
Get:4 http://azure.archive.ubuntu.com/ubuntu jammy/universe amd64 mesa-utils-bin amd64 8.4.0-1ubuntu1 [53.8 kB]
Get:5 http://azure.archive.ubuntu.com/ubuntu jammy/universe amd64 mesa-utils amd64 8.4.0-1ubuntu1 [1065 kB]
Fetched 1165 kB in 1s (837 kB/s)
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/m/mesa/libegl-mesa0_22.0.5-0ubuntu0.1_amd64.deb  404  Not Found [IP: 40.119.46.219 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.
````
@o01eg any ideas?

--- a/.github/workflows/snapcraft.yml
+++ b/.github/workflows/snapcraft.yml
@@ -36,6 +36,7 @@ jobs:
         sudo snap install --dangerous ${{ steps.snapcraft.outputs.snap }}
     - name: Install dependencies for test launch
       run: |
+        sudo apt update -y
         sudo apt install -y mesa-utils libgl1-mesa-dri libglx-mesa0
     - name: Test freeorion launch
       uses: GabrielBB/xvfb-action@v1.5

